luk@luk-pc:~/github/ext4_to_f2fs/linux/boot/ramdisk$ diff -Naur fstab.capri_ss_s2vep fstab.capri_ss_s2vep.f2fs 
--- initrd\fstab.capri_ss_s2vep.orig	2014-08-06 22:09:33.062598353 +0200
+++ initrd\fstab.capri_ss_s2vep		2014-08-06 22:09:33.062598353 +0200
@@ -4,12 +4,12 @@
 # specify MF_CHECK, and must come before any filesystems that do specify MF_CHECK
 
 #<src>					<mnt_point>		<type>	<mnt_flags and options>								<fs_mgr_flags>
-/dev/block/mmcblk0p19			/system			ext4	ro,errors=panic									wait
+#/dev/block/mmcblk0p19			/system			ext4	ro,errors=panic									wait
 /dev/block/mmcblk0p17			/efs			ext4	nosuid,nodev,noatime,noauto_da_alloc,discard,journal_async_commit,errors=panic	wait,check
-/dev/block/mmcblk0p18			/cache			ext4	nosuid,nodev,noatime,noauto_da_alloc,discard,journal_async_commit,errors=panic	wait,check
+#/dev/block/mmcblk0p18			/cache			ext4	nosuid,nodev,noatime,noauto_da_alloc,discard,journal_async_commit,errors=panic	wait,check
 
 # data partition must be located at the bottom for supporting device encryption
-/dev/block/mmcblk0p21			/data			ext4	nosuid,nodev,noatime,noauto_da_alloc,discard,journal_async_commit,errors=panic	wait,check,encryptable=footer
+#/dev/block/mmcblk0p21			/data			ext4	nosuid,nodev,noatime,noauto_da_alloc,discard,journal_async_commit,errors=panic	wait,check,encryptable=footer
 
 # vold-managed volumes ("block device" is actually a sysfs devpath)
 /devices/platform/sdhci.3/mmc_host/mmc2	/storage/sdcard1	auto	defaults									voldmanaged=sdcard1:auto

